### PR TITLE
Add OpenSSL dependency to Kent_tools 418 easyconfig to fix a link error

### LINKED ISSUE #13988: SSL link error for Kent_tools 2021a
```
make[3]: Entering directory `/dev/shm/Kent_tools/418/GCC-10.3.0/userApps/kent/src/hg/ratStuff/mafSpeciesSubset'
gcc -O2 -ftree-vectorize -march=native -fno-math-errno -fcommon -O2 -ftree-vectorize -march=native -fno-math-errno -std=c99 -Wall -Wformat -Wimplicit -Wreturn-type -Wuninitialized -std=c99 -Wall -Wformat -Wimplicit -Wreturn-type -Wuninitialized -D_FILE_OFFSET_BITS=64 -D
_LARGEFILE_SOURCE -D_GNU_SOURCE -DMACHTYPE_local -DUSE_HIC   -Wall -Wformat -Wimplicit -Wreturn-type -Wuninitialized -I../inc -I../../inc -I../../../inc -I../../../../inc -I../../../../../inc -I../../../htslib -I/apps/eb/2020b/skylake/software/libpng/1.6.37-GCCcore-10.3
.0/include/libpng16 -I/apps/eb/2020b/skylake/software/zlib/1.2.11-GCCcore-10.3.0/include -I/usr/include/freetype2 -DUSE_FREETYPE -I/include -I/apps/eb/2020b/skylake/software/libpng/1.6.37-GCCcore-10.3.0/include/libpng16  -o mafSpeciesSubset.o -c mafSpeciesSubset.c
cd pslCheck && echo pslCheck && make
pslCheck
true /dev/shm/Kent_tools/418/GCC-10.3.0/userApps/bin/liftOverMerge
true /dev/shm/Kent_tools/418/GCC-10.3.0/userApps/bin/autoSql
make[3]: Leaving directory `/dev/shm/Kent_tools/418/GCC-10.3.0/userApps/kent/src/hg/autoSql'
cd pslDropOverlap && echo pslDropOverlap && make
pslDropOverlap
make[3]: Leaving directory `/dev/shm/Kent_tools/418/GCC-10.3.0/userApps/kent/src/hg/liftOver'
make[3]: Entering directory `/dev/shm/Kent_tools/418/GCC-10.3.0/userApps/kent/src/hg/pslCDnaFilter'
echo '/* GENERATED CODE, DO NOT EDIT */' > usage.msg.tmp
cd pslFilter && echo pslFilter && make
pslFilter
/dev/shm/Kent_tools/418/GCC-10.3.0/userApps//bin/stringify usage.txt >> usage.msg.tmp
echo '/* GENERATED CODE, DO NOT EDIT */' > algo.msg.tmp
/dev/shm/Kent_tools/418/GCC-10.3.0/userApps//bin/stringify: error while loading shared libraries: libssl.so.1.1: cannot open shared object file: No such file or directory
make[3]: Entering directory `/dev/shm/Kent_tools/418/GCC-10.3.0/userApps/kent/src/hg/ratStuff/mafToSnpBed'
gcc -O2 -ftree-vectorize -march=native -fno-math-errno -fcommon -O2 -ftree-vectorize -march=native -fno-math-errno -std=c99 -Wall -Wformat -Wimplicit -Wreturn-type -Wuninitialized -std=c99 -Wall -Wformat -Wimplicit -Wreturn-type -Wuninitialized -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_GNU_SOURCE -DMACHTYPE_local -DUSE_HIC   -Wall -Wformat -Wimplicit -Wreturn-type -Wuninitialized -I../inc -I../../inc -I../../../inc -I../../../../inc -I../../../../../inc -I../../../htslib -I/apps/eb/2020b/skylake/software/libpng/1.6.37-GCCcore-10.3.0/include/libpng16 -I/apps/eb/2020b/skylake/software/zlib/1.2.11-GCCcore-10.3.0/include -I/usr/include/freetype2 -DUSE_FREETYPE -I/include -I/apps/eb/2020b/skylake/software/libpng/1.6.37-GCCcore-10.3.0/include/libpng16  -o mafToSnpBed.o -c mafToSnpBed.c
make[3]: Entering directory `/dev/shm/Kent_tools/418/GCC-10.3.0/userApps/kent/src/hg/genePredCheck'
gcc -O2 -ftree-vectorize -march=native -fno-math-errno -fcommon -O2 -ftree-vectorize -march=native -fno-math-errno -std=c99 -Wall -Wformat -Wimplicit -Wreturn-type -Wuninitialized -std=c99 -Wall -Wformat -Wimplicit -Wreturn-type -Wuninitialized -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_GNU_SOURCE -DMACHTYPE_local -DUSE_HIC   -Wall -Wformat -Wimplicit -Wreturn-type -Wuninitialized -I../inc -I../../inc -I../../../inc -I../../../../inc -I../../../../../inc -I../../htslib -I/apps/eb/2020b/skylake/software/libpng/1.6.37-GCCcore-10.3.0/include/libpng16 -I/apps/eb/2020b/skylake/software/zlib/1.2.11-GCCcore-10.3.0/include -I/usr/include/freetype2 -DUSE_FREETYPE -I/include -I/apps/eb/2020b/skylake/software/libpng/1.6.37-GCCcore-10.3.0/include/libpng16  -o genePredCheck.o -c genePredCheck.c
make[3]: *** [usage.msg] Error 127
make[3]: *** Waiting for unfinished jobs....
```

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

edit: fixes #13988

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 5 out of 5 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/1dcd57a3228274ae4581f7892af5b150 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/k/Kent_tools/Kent_tools-418-GCC-10.3.0.eb [hunk 1]
```diff
@@ -21,6 +21,7 @@ dependencies = [
     ('libpng', '1.6.37'),
     ('zlib', '1.2.11'),
     ('util-linux', '2.36'),
+    ('OpenSSL', '1.1.1k'),
```
Comment (boegel) [inline easybuild/easyconfigs/k/Kent_tools/Kent_tools-418-GCC-10.3.0.eb:24]: @verdurin Isn't the intention here to use the OpenSSL link wrapper?

```python
    ('OpenSSL', '1.1.', '', True),

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/k/Kent_tools/Kent_tools-418-GCC-10.3.0.eb (hunk 1 above)
Comment (verdurin) [reply easybuild/easyconfigs/k/Kent_tools/Kent_tools-418-GCC-10.3.0.eb]: @boegel yes, that was the intention - and I've made the change, ignoring that deliberate typo to show I was (eventually) paying attention...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3118.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/75b7405bfae2bbded910a58ae7e37051 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3504.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/d0a7e78d838d830b1b5d415e255c9996 for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @verdurin!

**Pull request merged**